Tasks/CopyFilesOverSSHV0/sshhelper.ts

```diff
@@ -113,19 +97,37 @@ export class SshHelper {
      * @param dest, folders will be created if they do not exist on remote server
      * @returns {Promise<string>}
      */
-    uploadFile(sourceFile: string, dest: string) : Q.Promise<string> {
+    async uploadFile(sourceFile: string, dest: string) : Promise<string> {
+        if (process.platform === 'win32') {
```

Comment:
Could you please move it to fuction like in scp? https://github.com/spmjs/node-scp2/blob/master/lib/client.js#L319